lib/logster/middleware/viewer.rb

```diff
@@ -35,7 +35,7 @@ def call(env)
             if status == 200
               body = [compile_hbs(body.to_path, type, name)]
               headers['Content-Type'] = 'application/javascript'
-              headers['Content-Length'] = body.first.bytesize
+              headers['Content-Length'] = body.first.bytesize.to_s
```

Comment:
double checked that this is also what `Rack::File` does: https://github.com/rack/rack/blob/5559676e7b5a3107d39552285ce8b714b672bde6/lib/rack/file.rb#L101